nfts/mainnet.json

```diff
@@ -0,0 +1,4 @@
+{
+  "networkId": 0,
+  "nfts": []
```

Comment:
Why does the mainnet.json have a different structure than the testnet.json?

Reply:
oops forgot to update it :/ mainnet should be the same as testnet